app/components/ui/FlightPauseIndicator.tsx

```diff
@@ -0,0 +1,63 @@
+import { Icon } from "@thorium/ui/Icon";
+import { useFlightStatus } from "@thorium/hooks/useFlightStatus";
+
+/**
```

Comment:
This comment is unnecessarily verbose for a relatively small, self-explanatory component. Did an AI write this?